src/main/java/sx/blah/discord/handle/impl/obj/Guild.java

```diff
@@ -779,6 +778,18 @@ public IEmoji getEmojiByName(String name) {
 	}
 
 	@Override
+	public IEmoji addEmoji(String name, BufferedImage image){
```

Comment:
I don't know if it's a good idea to accept BufferedImage because I'm not sure if headless jre installations  will have that class available.